README.md

```diff
@@ -8,26 +8,60 @@ DataLad registry -- work in progress
 
 ### NEW Development setup
 
+#### Preface
+
+We fully test Datalad-Registry with Podman. Nevertheless, you should be able to launch
+a Datalad-Registry instance using Docker with little to no deviation from this guide.
+
+#### Prerequisites
+
+The following dependencies are needed in a system in order to test and develop
+Datalad-Registry:
+
+  * [git](https://git-scm.com/downloads)
+  * [git-annex](https://git-annex.branchable.com/install/)
+  * [Podman](https://podman.io/docs/installation)
+  * [podman-compose](https://github.com/containers/podman-compose)
+
+    We strongly recommend installing `podman-compose` and other Python dependencies in
+    a [Python virtual environment](https://docs.python.org/3/library/venv.html) for this
+    project.
+
 #### To run tests
 
-On Debian systems install necessary for Python PostgreSQL libs dependencies:
+1. Setup
+
+    1. On Debian systems, install the necessary dependencies for Python PostgreSQL libs:
+
+       `sudo apt-get install libpq-dev python3-dev`
```

Comment:
```suggestion
       `sudo apt-get install podman libpq-dev python3-dev`
```

since podman is needed?

Reply:
I am consolidating all the shared dependencies, for testing, development, and eventually production, into the  the "Prerequisites" section. Since I provided `Podman` as a dependency in the "Prerequisites" section, I am not going to install it in the testing setup in particular.

This arrangement will work better particularly for people who uses the same machine for both development and testing.

I am currently modifying the "To development" section.

I will add a production section once https://github.com/datalad/datalad-registry/issues/305 is resolved.